cabal-override.project

```diff
@@ -4,32 +4,9 @@ package proto3-suite
 packages: .
 
 -- dhall expects aeson < 2.2
-constraints: aeson < 2.2
```

Comment:
I think we can generate the constraints from the nixpkgs set using `cabal freeze`. That way we only need to track the package override which uses unrelease source, and the build should keep on working in the future.

Moreover, we should put these bounds in the monocle.cabal.